From: Aristeu Rozanski <arozansk@redhat.com>
Date: Mon, 12 May 2008 11:32:28 -0400
Subject: [edac] k8_edac: fix typo in user visible message
Message-id: 20080512153228.GD10306@redhat.com
O-Subject: [RHEL5 PATCH] k8_edac: fix typo in user visible message
Bugzilla: 446068
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Alan Cox <alan@redhat.com>
RH-Acked-by: Jarod Wilson <jwilson@redhat.com>
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=446068

This patch fixes a k8_edac error message.

The patch is upstream in bluesmoke project (amd64_edac and k8_edac)

diff --git a/drivers/edac/k8_edac.c b/drivers/edac/k8_edac.c
index 286d8ee..ad3bf65 100644
--- a/drivers/edac/k8_edac.c
+++ b/drivers/edac/k8_edac.c
@@ -1491,7 +1491,7 @@ static inline void decode_gart_tlb_error(struct mem_ctl_info *mci,
 	ec_tt = (err_code >> 2) & 0x03UL;
 	ec_ll = (err_code >> 0) & 0x03UL;
 	k8_mc_printk(mci, KERN_ERR,
-		     "GART TLB errorr: transaction type(%s), "
+		     "GART TLB error: transaction type(%s), "
 		     "cache level(%s)\n", tt_msgs[ec_tt], ll_msgs[ec_ll]);
 }
 
